cpp/tests/CMakeLists.txt

```diff
@@ -1,19 +1,17 @@
-#=============================================================================
+# =============================================================================
 # cmake-format: off
 # SPDX-FileCopyrightText: Copyright (c) 2018-2025, NVIDIA CORPORATION.
 # SPDX-License-Identifier: Apache-2.0
 # cmake-format: on
```

Comment:
For some reason after rebasing to fix the merge conflicts caused by #7373, `pre-commit` reformatted this whole file. I presume this is due to the addition of this line here (though it's not clear to me why that PR didn't fail due to pre-commit checks then). :shrug:

Anyway, the only real change in this file was removing the cmake setup for the removed test file, I didn't touch anything else myself.